<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="TABLE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>â„–</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Random Value</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="BUTTON">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>BUTTON</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Parametrs">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Parametrs</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_Sigma">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_n">
       <property name="text">
        <string>n =</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_k">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_a">
       <property name="text">
        <string>a =</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_a">
       <property name="text">
        <string>0.5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_n">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_k">
       <property name="text">
        <string>k = </string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_Sigma">
       <property name="text">
        <string>Sigma = </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_Informations">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Informations</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>23</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>fn(y) = (y/Sigma^2)exp{-y^2/2Sigma^2}; y&gt;= 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Range">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>R^ (Range) =</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_D">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>D (max|Fj-F^j|)=</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_DS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>|D - S^2| =</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_maxDensityH">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>max|fj - nj/n*dj|</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_R">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>R0 = </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_FR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>F(R) =</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Hypothesis">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hypothesis:</string>
     </property>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="GRAPH" native="true">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>421</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CLEAN">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CLEAN</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="GRAPH_1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="TABLE_1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>401</width>
      <height>131</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>zj</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>fj</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>nj/n*dj</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>|fj-nj/n*dj|</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="TABLE_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>370</y>
      <width>421</width>
      <height>131</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Smpl mean</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Math expectation</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Error</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Smpl Dispersion</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dispersion</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Error</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Smpl Median</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Median</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Error</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="TABLE_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Begin</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>End</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Q</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Begin">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>160</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_End">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>190</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1161</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu8_Variant">
    <property name="title">
     <string>8 Variant</string>
    </property>
   </widget>
   <addaction name="menu8_Variant"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
